### PR TITLE
docs: add note about the nixos service defaulting to sqlite

### PR DESCRIPTION
**Pull request by tamara-schmitz**

### CODE REVIEW

File: docs/deploying/nixos.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ The `flake.nix` and `default.nix` do not currently provide a NixOS module (contr
 welcome!), so [`services.matrix-conduit`][module] from Nixpkgs can be used to configure
 conduwuit.
 
+Beware! The module defaults to SQLite as a database backend. Conduwuit dropped SQLite support
```
Comment (girlbossceo) [inline docs/deploying/nixos.md:42]: Can this be put in a dedicated header to be more visible?

```suggestion
### Conduit NixOS Config Module and SQLite

Beware! The module defaults to SQLite as a database backend. Conduwuit dropped SQLite support
```

Comment (tamara-schmitz) [reply docs/deploying/nixos.md:44]: hm. then with this lemme also rephrase my sentence a bit

Comment (tamara-schmitz) [reply docs/deploying/nixos.md:44]: should i squash it all down into one commit

**Comment by girlbossceo** [comment]
I'm planning on merging this + another couple PRs here after we merge our huge async database changes to avoid jumping through rebase mess for everyone. It will be couple days, but if something comes up and delays it then I'll merge it anyways.

**Comment by girlbossceo** [comment]
Merged in https://github.com/girlbossceo/conduwuit/commit/2b2793fac6b2384217e70424dbd16752060dec14

**Pull request merged**